conf/site-local.conf

```diff
@@ -29,6 +29,9 @@ server {
                 expires off;
                 add_header Cache-Control "no-store";
                 add_header X-Frame-Options "DENY";
+                add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
+                add_header X-Content-Type-Options nosniff;
+                add_header Referrer-Policy 'no-referrer';
```

Comment:
Do we not use the referrer for CORS request for _GetUserData_ with Doppler Cookie?

Reply:
I will test this local, pointing to production as we spoke, to test further. There is an option to send the domain only for CORS, if needed I can use that.

Reply:
I will test this further, thanks!

Reply:
I've researched this a bit more, and `origin-when-cross-origin` seems to be the best fit.
Some examples:

![image](https://user-images.githubusercontent.com/2439363/113612007-a4eec580-9625-11eb-96a6-8b76e210d6bc.png)

We will not have troubles using the `referer` or `origin` header for inside the same site, and in a different site, only the origin will be sent as both the `origin` and `referer` headers.

Reply:
👍